tasks/build_client.yml

```diff
@@ -0,0 +1,14 @@
+---
+# First ensure that nodejs is installed into the venv by running common_startup.
+- name: Execute common_startup.sh
```

Comment:
Thus far I've tried to recreate `common_startup.sh` in the role rather than call it directly since it gives the role user more control and is more Ansible-ic. I can do this for the client build if you want to focus elsewhere (I don't want to waste your time on my inanity). Bonus: [tasks are already written](https://github.com/galaxyproject/usegalaxy-playbook/blob/master/roles/usegalaxy_cvmfs/tasks/_inc_client_build.yml).

Reply:
#52

Reply:
Next time could you just add to my branch? Or PR to my branch if you don't want to add directly? That way I don't have to go back and re-edit this PR to make it merge conflict safe?